src/index.js

```diff
@@ -150,14 +150,17 @@ const processor = postcss.plugin('postcss-modules-scope', function(options) {
     let exportedNames = Object.keys(exports);
     if (exportedNames.length > 0) {
       css.append(postcss.rule({
-        selector: `:export`,
-        nodes: exportedNames.map(exportedName => postcss.decl({
+        selector: `:export`
+      }));
+      let exportRule = css.nodes[css.nodes.length - 1];
```

Comment:
Sorry for late reply. I just didn't notice that method and @just-boris 's way of doing looks better absolutely.